i18n/it/cosmic_settings.ftl

```diff
@@ -241,195 +245,195 @@ x-hours = { $number ->
 
-workspaces-orientation = Orientamento Spazi di Lavoro
+workspaces-orientation = Orientamento spazi di lavoro
     .vertical = Verticale
     .horizontal = Orizzontale
 
 ## Networking: Wired
 
-wired = Cablata
-    .desc = Connessione cablata, profili di connessione
+wired = Via cavo
+    .desc = Connessione via cavo, profili di connessione
 
 ## Networking: Online Accounts
 
-online-accounts = Account Online
-    .desc = Aggiungi account, IMAP e SMTP, login aziendali
+online-accounts = Account online
+    .desc = Aggiungi account IMAP, SMTP e enterprise
 
 ## Time & Language
 
-time = Ora e Lingua
+time = Orario e linguaggio
     .desc = N/A
 
 time-date = Data e ora
-    .desc = Fuso orario, impostazioni orologio automatiche, e alcuni formati ora.
+    .desc = Fuso orario, impostazioni orologio automatico e formattazione orario
     .auto = Imposta automaticamente
 
 time-zone = Fuso orario
     .auto = Fuso orario automatico
-    .auto-info = Richiede geolocalizzazione e accesso a internet
+    .auto-info = Richiede i servizi di posizione e l'accesso a internet
 
 time-format = Formato data e ora
-    .twenty-four = formato 24 ore
+    .twenty-four = Formato 24 ore
     .first = Primo giorno della settimana
 
-time-region = Regione e Lingua
-    .desc = Formato date, ora e numeri in base alla tua regione
+time-region = Lingua e località
+    .desc = Formattazione data, ora e numeri basata sulla località
 
 ## Sound
 
 sound = Suono
     .desc = N/A
 
-sound-output = Uscita
-    .volume = Volume in uscita
-    .device = Dispositivo in uscita
-    .level = Livello in uscita
+sound-output = Output
+    .volume = Volume di output
+    .device = Dispositivo di output
+    .level = Livello di output
     .config = Configurazione
     .balance = Bilanciamento
 
-sound-input = Ingresso
-    .volume = Volume in ingresso
-    .device = Dispositivo in ingresso
-    .level = Livello in ingresso
+sound-input = Input
+    .volume = Volume di input
+    .device = Dispositivo di input
+    .level = Livello di input
 
-sound-alerts = Notifiche
-    .volume = Volume notifiche
-    .sound = Suono notifiche
+sound-alerts = Suoni di sistema
+    .volume = Volume dei suoni di sistema
+    .sound = Tipo suono di sistema
 
 sound-applications = Applicazioni
-    .desc = Volume applicazioni e impostazioni
+    .desc = Volumi delle applicazioni
 
 ## System
 
-system = Sistema e Account
+system = Sistema e account
 
 ## System: About
 
-about = Descrizione
-    .desc = Nome dispositivo, informazioni hardware, predefiniti sistema operativo.
+about = Informazioni
+    .desc = Nome dispositivo, informazioni hardware e impostazioni predefinite di sistema
 
 about-device = Nome dispositivo
-    .desc = Questo nome appare a altri dispositivi in rete o via bluetooth.
+    .desc = Ai dispositivi bluetooth, o a una rete, apparirà questo nome
 
 about-hardware = Hardware
     .model = Modello hardware
-    .memory = RAM
+    .memory = Memoria
```

Comment:
I would keep the reference to RAM here. A less tech savy user could get confused with mass storage, which is also called "memoria" sometimes.

Reply:
Wait, I don't remember changing that...

By the way, if "memoria" is badly used, you can thank our beloved smartphones and commercial bullshit.

Reply:
I was working with some people who thought the whole desktop is the memory and they the monitor for the was the computer 😅 so better be precise in cases such as this.

Reply:
HAHAHA I've yet to experience such pain